frontend/src/components/transactions/ListOfTransactions.js

```diff
@@ -1,55 +1,52 @@
 import React from "react";
-
 class Transactions extends React.Component {
   state = {
         transactionList : [{
-                                 id: 1,
-                                 firstname: 'Robin',
-                                 lastname: 'Wieruch',
-                                 amount: 1988,
-                                 date: '10/12/2018',
+                                 ID: 1,
+                                 Name: 'Robin',
```

Comment:
Why have you removed the firstname & lastname?

Reply:
In login page we just had Name. User is not going to enter any other data.